posthog/apps.py

```diff
@@ -48,18 +48,19 @@ def ready(self):
                     {"git_rev": get_git_commit_short(), "git_branch": get_git_branch()},
                 )
 
-            try:
-                user = User.objects.filter(last_login__isnull=False).order_by("-last_login").first()
-            except:
-                user = None
```

Comment:
Shouldn't this `try/except` be catching that error? Also, the error happens when loading `posthog_team`, so this is kinda weird, right?

Reply:
The error is caused by `local_api_key = get_self_capture_api_token(user)`

Good point about `try/except`, though. I simplified the PR 6364cf1031

Reply:
Looks like it was introduced in https://github.com/PostHog/posthog/pull/27669. I'll leave to @skoob13 and @pauldambra to review

Reply:
Actually, it looks like 6364cf1031744f2d085d001e4001959fdddfbf01 would break the fallback behavior:

https://github.com/PostHog/posthog/blob/6364cf1031744f2d085d001e4001959fdddfbf01/posthog/utils.py#L500-L515

Reply:
Reverted back to the original behavior because I think bypassing any user config is a more appropriate fix for the migration context. Waiting for additional input before landing, though.